interface/elements/msc-profile-view/msc-profile-view.js

```diff
@@ -68,7 +68,7 @@ Polymer({
         var pathOfKey = 'C:\\Users\\' + username + '\\AppData\\Roaming\\Musicoin\\keystore';
       } else if (platform.includes("darwin")) {
         var pathOfKey = '/Users/' + username + '/Library/Musicoin/keystore';
-      } else if (platform.includes()){ //linux
+      } else if (platform.includes("linux")){ //linux
```

Comment:
What I thought here was that in other systems (BSD, Solaris, etc.) the key store *might* be in the same place. Better to make it explicit I guess.